editor/src/clj/editor/engine/native_extensions.clj

```diff
@@ -337,9 +356,12 @@
         (map (juxt #(resource-node-upload-path % evaluation-context) identity))
         (extension-resource-nodes project evaluation-context platform)))
 
-(defn has-extensions? [project evaluation-context]
-  (not (empty? (merge (extension-roots project evaluation-context)
-                      (global-resource-nodes-by-upload-path project evaluation-context)))))
```

Comment:
It's a miracle this code worked. `extension-roots` returns `seq` or `nil`, and `global-resource-nodes-by-upload-path` returns a map. Invoking `merge` on a seq and a map just conjed a map to the seq and the result was non-empty, so it worked "correctly".